### DISCUSSION TITLE
Apple Watch app

### DISCUSSION THREAD

**Discussion by Bammede** [organicmaps/organicmaps#3150] [Ideas]
Apple Watch app and navigation

**Comment by biodranik** [comment]
Why do you need it? :)

**Reply by dz86** [reply]
Organic map, beautiful for Hiking, and no need phone
Please do watch os version

**Reply by blackrez** [reply]
+1 for support Apple watch. The app is great on Iphone but I have to use plan on apple watch.

**Reply by louwers** [reply]
I don't think the maps from Apple are rendered on the watch itself. The Apple Watch does not have a GPU that is accessible by application developers. So maps would need to be rendered somewhere else. For Organic Maps the only place that would make sense is a companion app running on an iPhone. But you would still need your phone, so that is probably not worth the effort.

Until Apple ships an Apple Watch with Metal support I don't see this happening. Please correct me if I am wrong.

**Reply by ahrorkosimi** [reply]
This is a very needed feature as I also don't want to take my phone while trail running and hiking with a lot of weight in my back. Wikilock and WorkOutDoors let you download map locally on apple watch. But if organic maps had it that would be teriffic. One more reason not to be tied to phone.

**Reply by ryanjkemper** [reply]
I also enjoy using maps on my watch, even if my phone is in my pocket while biking or using a scooter, so a companion app running on the phone wouldn’t be a big deal for this use case

**Reply by sacameron** [reply]
+1 support for Apple Watch. Offline maps on Watch (stored locally on Watch, completely independent from iPhone) is ripe for opportunity as this doesn't exist natively on the platform. There are some 3rd party apps that feature this, but I prefer Organic Maps on iPhone and would love to have the same app on my Watch as well.

Uses: hiking, anywhere off the grid. It would add value to my Apple Watch and help it do something it can't do natively. Especially if it synergizes with Organic Maps on my iPhone which I already use. I just would love to have a map of my general region / state saved for offline use on my watch.

**Comment by Bammede** [comment]
maps & navigation run independently from iPhone.

**Comment by haozes** [comment]
+1 Apple Watch Ultra has enough battery to make a long trip. If it works on watch it will be great!!!

**Comment by slidingshade** [comment]
Not sure if I need full-featured version for apple watch but it would be great to have at least turn-by-turn directions on my watch.

**Comment by cvanelteren** [comment]
Would also love this feature, it would rid me of using the default Apple maps.

**Comment by xairoo** [comment]
Mapy can be used with the Apple Watch. I don’t like Mapy but yeah, it works with the watch - so why shouldn’t it work for Organic Maps?
